DuckDuckGo/TabViewController.swift

```diff
@@ -1147,7 +1147,12 @@ extension TabViewController: WKNavigationDelegate {
         DispatchQueue.main.async { [weak self] in
             guard let webView = self?.webView,
                   webView.bounds.height > 0 && webView.bounds.width > 0 else { completion(nil); return }
-            UIGraphicsBeginImageContextWithOptions(webView.bounds.size, false, UIScreen.main.scale)
+            
+            let size = CGSize(width: webView.frame.size.width,
+                              height: webView.frame.size.height - webView.scrollView.contentInset.top - webView.scrollView.contentInset.bottom)
+            
+            UIGraphicsBeginImageContextWithOptions(size, false, UIScreen.main.scale)
+            UIGraphicsGetCurrentContext()?.translateBy(x: 0, y: -webView.scrollView.contentInset.top)
```

Comment:
we're only interested in the bit of the screen currently visible - without this we end up with potentially blank space